test/torchaudio_unittest/transforms/transforms_test.py

```diff
@@ -180,6 +180,65 @@ def test_mfcc_norms(self):
 
         self.assertEqual(torch_mfcc_norm_none, norm_check)
 
+    def test_lfcc_defaults(self):
+        """Check default settings for LFCC transform.
+        """
+        sample_rate = 16000
+        audio = common_utils.get_whitenoise(sample_rate=sample_rate)
+
+        n_lfcc = 40
+        n_filter = 128
+        lfcc_transform = torchaudio.transforms.LFCC(sample_rate=sample_rate,
+                                                    n_filter=n_filter,
+                                                    n_lfcc=n_lfcc,
+                                                    norm='ortho')
+        torch_lfcc = lfcc_transform(audio)  # (1, 40, 81)
+        self.assertEqual(torch_lfcc.dim(), 3)
+        self.assertEqual(torch_lfcc.shape[1], n_lfcc)
+        self.assertEqual(torch_lfcc.shape[2], 81, torch_lfcc.shape[2])
```

Comment:
``torch_lfcc.shape[2]`` is duplicated. Let's change it to:
```
self.assertEqual(torch_lfcc.shape[2], 81)
```

Reply:
Resolved in 51aced6.